src/pbft/pbft_config.h

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#include "libbyz/libbyz.h"
+#include "libbyz/pbft_assert.h"
+#include "pbft/pbft_deps.h"
+
+namespace pbft
+{
+  static char* service_mem = 0;
+
+  class PbftConfig
```

Comment:
Minor: `AbstractPbftConfig` to keep it consistent with other interface classes in CCF?

Reply:
AbstractPBFTConfig?